infrastructure/terraform/worker-manager-service.tf

```diff
@@ -0,0 +1,48 @@
+resource "random_string" "worker_manager_access_token" {
+  length           = 65
+  override_special = "_-"
+}
+
+module "worker_manager_secrets" {
+  source            = "modules/service-secrets"
+  project_name      = "taskcluster-worker-manager"
+  disabled_services = "${var.disabled_services}"
+
+  secrets = {
+    TASKCLUSTER_CLIENT_ID    = "static/taskcluster/worker-manager"
```

Comment:
aws creds as well?

Reply:
Yep, we can make that work once it needs them.